prow/plugins/cherrypickunapproved/cherrypick-unapproved_test.go

```diff
@@ -88,21 +89,28 @@ type fakePruner struct{}
 
 func (fp *fakePruner) PruneComments(shouldPrune func(github.IssueComment) bool) {}
 
-func makeFakePullRequestEvent(action github.PullRequestEventAction, branch string) github.PullRequestEvent {
-	return github.PullRequestEvent{
+func makeFakePullRequestEvent(action github.PullRequestEventAction, branch string, changes json.RawMessage) github.PullRequestEvent {
+	event := github.PullRequestEvent{
 		Action: action,
 		PullRequest: github.PullRequest{
 			Base: github.PullRequestBranch{
 				Ref: branch,
 			},
 		},
 	}
+
+	if changes != nil {
+		event.Changes = changes
+	}
```

Comment:
Nit: You could have just put this into the event unconditionally

Reply:
You're right, I forgot to modify this after changing the logic around how we pass changes around.